src/convertToObject.js

```diff
@@ -6,7 +6,32 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const mainObj = {};
```

Comment:
Consider using a more descriptive variable name instead of `mainObj`. While it is clear that it's the main object for the function, it could be named to reflect its purpose more clearly, such as `styleObject`.